### PR TITLE
feat: create webpack confing for react

### PR DESCRIPTION
**Pull request by SergeyIankowski**
I created webpack config for our project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+npm run build
+```
+
+or
+
+```
+yarn build
+```
+
+to bundle your application
```
Comment (sekaa4) [inline README.md]: we have readme file in the develop branch, we dont need it here

File: src/components/App.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function App() {
+  return <h1>Hello world</h1>
+}
+
+export default App;
```
Comment (sekaa4) [inline src/components/App.tsx]: empty string needs here in the end file

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+};
+
+module.exports = () => {
+  if (isProduction) {
+    config.mode = "production";
+  } else {
+    config.mode = "development";
+  }
+  return config;
+};
```
Comment (sekaa4) [inline webpack.config.js]: empty string needs here in the end file

**Review by sekaa4 — COMMENTED** [review]
we need eslint config and pritier too
 https://www.youtube.com/watch?v=ZXW6Jn6or1w&t=247s

eslint try add airbnb config if u can.

**Review by sekaa4 — APPROVED** [review]

**Review by ImmelstronDev — APPROVED** [review]

**Pull request merged**